content/security/authorization.md

```diff
@@ -234,16 +234,16 @@ With this in place, we can define the `createForUser()` method on the `CaslAbili
 ```typescript
 type Subjects = InferSubjects<typeof Article | typeof User> | 'all';
 
-export type AppAbility = Ability<[Action, Subjects]>;
+type AppAbility = MongoAbility<[Action, Subjects]>;
 
 @Injectable()
 export class CaslAbilityFactory {
   createForUser(user: User) {
-    const { can, cannot, build } = new AbilityBuilder<
-      Ability<[Action, Subjects]>
-    >(Ability as AbilityClass<AppAbility>);
+    const { can, cannot, build } = new AbilityBuilder<AppAbility>(
+      createMongoAbility,
```

Comment:
This guide isn't mongo specific so why "createMongoAbility"?

Reply:
see here: https://github.com/stalniy/casl/blob/master/packages/casl-ability/src/Ability.ts#L9,

If I am honest, the name "Ability" was much better, while "MongoAbility" which is very confusing.